cmd/dep/init.go

```diff
@@ -237,6 +238,15 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 	packages := make(map[string]bool)
 	notondisk := make(map[gps.ProjectRoot]bool)
 	ondisk := make(map[gps.ProjectRoot]gps.Version)
+
+	printNewDeP := func(pr gps.ProjectRoot, sm *gps.SourceMgr) {
+		message := "Cloning"
```

Comment:
Actually, might be better to switch out the word entirely - "cloning" applies to git, but git isn't all we're dealing with here. Maybe "Fetched latest of"? I think that's generic enough.